xml/MAIN-SBP-SLSA.xml

```diff
@@ -0,0 +1,1515 @@
+         </para>
+        </formalpara>
+       </listitem>
+       <listitem>
+        <formalpara>
+         <title>SUSE's processes</title>
+         <para>
+          Red team scans for vulnerabilities, build team patches build
+          environment. Machine identity is enforced, user isolation by policy.
+         </para>
+        </formalpara>
+       </listitem>
+      </itemizedlist>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>Access</term>
+     <listitem>
+      <itemizedlist>
+       <listitem>
+        <formalpara>
+         <title>SLSA L4 requirement</title>
+         <para>
+          Make sure that physical and/or remote access is rare and tightly
+          regulated. This includes logging and multi-party approval.
+         </para>
+        </formalpara>
+       </listitem>
+       <listitem>
+        <formalpara>
+         <title>SUSE's processes</title>
+         <para>
+          Multiparty approval in general is done as logging is done via SSH
+          auth key which is salted.
+         </para>
+        </formalpara>
+       </listitem>
+      </itemizedlist>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>Superusers</term>
+     <listitem>
+      <itemizedlist>
+       <listitem>
+        <formalpara>
+         <title>SLSA L4 requirement</title>
+         <para>
+          Restrict the number of admins allowed to override the above listed
+          criteria to the bare minimum. Whenever such interaction is required,
+          a second admin must give their approval.
+         </para>
+        </formalpara>
+       </listitem>
+       <listitem>
+        <formalpara>
+         <title>SUSE's processes</title>
+         <para>
+          Number of superusers in OBS is very small.
+         </para>
+        </formalpara>
+       </listitem>
+      </itemizedlist>
+     </listitem>
+    </varlistentry>
+   </variablelist>
+  </sect2>
+ </sect1>
+ <sect1 xml:id="sec-outlook">
+  <title>Outlook</title>
+
+  <para>
+   SUSE is up to a great start with claiming SLSA L4 compliance, as SLSA
+   requirements partly overlap with those of Common Criteria EAL4+. This meant
+   that several SLSA criteria were met by SUSE's supply chain processes right
+   from the start. The core part of this is our certified and proven build and
+   integration process which uses the Open Build Service technology.
+  </para>
+
+  <para>
+   Over the past few months, SUSE has been working on improving and tightening
+   processes and technologies in order to be able to claim full SLSA L4
+   compliance.
+  </para>
+
+  <para>
+   New to SUSE is providing information of the build environment
+   (<emphasis>provenance</emphasis>) to the outside world. So far, this kind of
+   information was kept internal, but as of now it will be delivered in the
+   <emphasis>in-toto</emphasis> format, allowing external parties to verify the
+   full build environment used for building all SUSE deliverables.
```

Comment:
```suggestion
   New to SUSE is providing the information of the build environment
   (<emphasis>provenance</emphasis>) to the outside world. So far, this kind of
   information has been kept internal, but now it will be delivered in the
   <emphasis>in-toto</emphasis> format, allowing external parties to verify the
   full build environment used for building all SUSE deliverables.
```